cmd/manager/main.go

```diff
@@ -178,14 +181,16 @@ func main() {
 		filterAPI,
 		authFactory,
 		scheme,
+		operatorNamespace,
```

Comment:
what do you say about making operatorNamespace a constant ("openshift-kmm")?  Why would a customer want to deploy it in a different one?

Reply:
I don’t think we can force the user to install into `openshift-KMM`, so I would leave this as is. It is populated automatically by the downwards API.